### ISSUE TITLE
Save default sort option for chapters

### ISSUE THREAD

**Issue by Xellerate005** [Aidoku/Aidoku#256]
Add an option for a default sorting method for specific or all mangas/comics, eg select sort by chapters or sort by date in settings for all comics/mangas to sort that way

**Comment by Skittyblock** [comment]
Library has sort options that save, sources have unique filters and don't have a way to set a default sort option (since it's different and dynamic for every source), but we can have an option to save the chapter sort method inside manga pages.

**Comment by Xellerate005** [comment]
yeah I've had an issue with some manga staring at chapter 50 instead on chapter 1, I then have to manually select sort by chapter to read from chapter 1, but it always changes back when I go to a different manga

**Comment by Skittyblock** [comment]
considering that manga now save the configured chapter sort/filter settings, i'm going to mark this as resolved
